gadget/gadgettest/examples.go

```diff
@@ -808,6 +808,40 @@ volumes:
         size: 1G
 `
 
+const SingleVolumeUC20GadgetYamlSeedFsLabelCaps = `
+volumes:
+  pc:
+    schema: gpt
+    bootloader: grub
+    structure:
+      - name: mbr
+        type: mbr
+        size: 440
+      - name: ubuntu-seed
+        role: system-seed
+        filesystem: vfat
+        filesystem-label: UBUNTU-SEED
```

Comment:
in our scenario this is still lower case but the disk one is not, so we need an update test for that scenario too, no?